woodwork/tests/utils/test_accessor_utils.py

```diff
@@ -101,6 +101,14 @@ def test_init_series_with_datetime(sample_datetime_series):
     assert isinstance(series.ww.logical_type, Datetime)
 
 
+def test_init_series_with_latlong(latlong_df_pandas):
```

Comment:
Is there a reason for testing this with only the pandas version of the fixture and not testing with Dask & Koalas?